db/migrate/20150709221755_create_logins.rb

```diff
@@ -1,4 +1,4 @@
-class CreateLogins < ActiveRecord::Migration
+class CreateLogins < ActiveRecord::Migration[4.2]
```

Comment:
This only works for Rails >= 5.0. Previous versions don't define the `[]` method on the `ActiveRecord::Migration` class.